web/src/pages/Route/components/Step1/MetaView.tsx

```diff
@@ -285,27 +285,44 @@ const MetaView: React.FC<RouteModule.Step1PassProps> = ({ disabled, form, isEdit
   )
 
   const ServiceSelector: React.FC = () => (
-    <Form.Item label={formatMessage({ id: 'page.route.service' })} tooltip="绑定服务（Service）对象，以便复用其中的配置。">
-      <Row>
-        <Col span={5}>
-          <Form.Item noStyle name="service_id">
-            <Select disabled={disabled}>
-              {/* TODO: value === '' means  no service_id select, need to find a better way */}
-              <Select.Option value="" key={Math.random().toString(36).substring(7)}>
-                {formatMessage({ id: "page.route.service.none" })}
-              </Select.Option>
-              {serviceList.map((item) => {
-                return (
-                  <Select.Option value={item.id} key={item.id}>
-                    {item.name}
-                  </Select.Option>
-                );
-              })}
-            </Select>
-          </Form.Item>
-        </Col>
-      </Row>
-    </Form.Item>
+    <React.Fragment>
+      <Form.Item label={formatMessage({ id: 'page.route.service' })} tooltip="绑定服务（Service）对象，以便复用其中的配置。">
```

Comment:
i18n

Reply:
Hi, this PR will fix this.

https://github.com/apache/apisix-dashboard/pull/1785/files#diff-48bf46c8113258031a14f9204041bb5ff51de312ef61c7686f41ea3499fe2c14R176